src/CMakeLists.txt

```diff
@@ -6,6 +6,13 @@ ADD_LIBRARY(redisai_obj OBJECT
     err.c
     script.c
     tensor.c
-    utils/alloc.c)
+    util/alloc.c
+    rmutil/sds.c
+    rmutil/args.c
```

Comment:
I had alloc before, I just changed the directory from `utils` to `util`. `sds` is a dependency for now, but I'm not using it at the moment, so conditional would work. It can stay for now I guess.